supervisor/gunicorn.conf

```diff
@@ -1,5 +1,5 @@
 [program:--DOMAIN_NAME--]
-command=envdir env bin/gunicorn --PROJECT--.wsgi -b unix:/var/www/%(program_name)s/gunicorn.sock
+command=envdir env bin/gunicorn --PROJECT--.wsgi -w 2 -b unix:/var/www/%(program_name)s/gunicorn.sock
```

Comment:
As this flag is in a config file, it's probably a good idea to use the verbose flag for clarity. `--workers`